BudgetBuddy/src/main/java/com/budgetbuddy/BudgetBuddyController.java

```diff
@@ -100,34 +100,38 @@ public String mngBill(Model model)
     @RequestMapping("/entry-form/save-exp")
     public String saveExp(Expense expense, Model model) {
         //set page to active
+        String returnValue;
         model.addAttribute("page", "entry");
 
         try {
             //save expense
             //ADD BREAK POINT HERE TO SEE SAVED DATA
             expenseService.save(expense);
+            returnValue = "mngexp";
         } catch (Exception e) {
             e.printStackTrace();
-            return "mngexp";
+            returnValue = "mngexp";
         }
-        return "mngexp";
+        return returnValue;
```

Comment:
It is best practice to only return once in a method.
https://www.baeldung.com/java-clean-code